### ISSUE TITLE
TypeError: utils_1.BindOption is not a function

### ISSUE THREAD

**Issue by socsieng** [TypeStrong/typedoc#1172]
- [x] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates

The typedoc cli returns the following error:

```
TypeError: utils_1.BindOption is not a function
```

## Expected Behavior

Expect `typedoc --version` to display version information

## Actual Behavior

```
/Users/***/node_modules/typedoc/dist/lib/output/renderer.js:134
    utils_1.BindOption('theme')
            ^

TypeError: utils_1.BindOption is not a function
    at Object.<anonymous> (/Users/***/node_modules/typedoc/dist/lib/output/renderer.js:134:13)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/***/node_modules/typedoc/dist/lib/output/plugins/AssetsPlugin.js:13:20)
    at Module._compile (module.js:653:30)
```

## Steps to reproduce the bug

```
npm install --global typedoc

typedoc --version
```

## Environment
  - Typedoc version: 0.16.3
  - Node.js version: 8.16.0
  - OS: macOS Catalina

**Comment by Gerrit0** [comment]
How on earth did all the tests pass with an error like that... looking into it. Sorry about that!

**Comment by Gerrit0** [comment]
Fixed in 0.16.4
